tox.ini

```diff
@@ -12,7 +12,7 @@ whitelist_externals =
     sed
 deps =
     -rtest-requirements.txt
-    ansible{env:ANSIBLE_VERSION:==2.1.0}
+    ansible{env:ANSIBLE_VERSION:==2.2.2}
```

Comment:
Is there a reason why we're setting this here? can we not just use the included ansible-bootstrap within osa or the openstack-ansible-tests repo to ensure this consistency?